storybook/pages/ActivityNotificationCommunityMembershipRequestPage.qml

```diff
@@ -58,11 +58,12 @@ SplitView {
                 valueRole: "value"
                 model: ListModel {
                          id: model
-                         ListElement { text: "Pending"; value: ActivityCenterStore.ActivityCenterMembershipStatus.Pending }
-                         ListElement { text: "Accepted"; value: ActivityCenterStore.ActivityCenterMembershipStatus.Accepted }
-                         ListElement { text: "Declined"; value: ActivityCenterStore.ActivityCenterMembershipStatus.Declined }
-                         ListElement { text: "AcceptedPending"; value: ActivityCenterStore.ActivityCenterMembershipStatus.AcceptedPending }
-                         ListElement { text: "DeclinedPending"; value: ActivityCenterStore.ActivityCenterMembershipStatus.DeclinedPending }
+
+                         ListElement { text: "Pending"; value: 1 } // ActivityCenterStore.ActivityCenterMembershipStatus.Pending }
+                         ListElement { text: "Accepted"; value: 2 } //ActivityCenterStore.ActivityCenterMembershipStatus.Accepted }
+                         ListElement { text: "Declined"; value: 3 } //ActivityCenterStore.ActivityCenterMembershipStatus.Declined }
+                         ListElement { text: "AcceptedPending"; value: 4 } //ActivityCenterStore.ActivityCenterMembershipStatus.AcceptedPending }
+                         ListElement { text: "DeclinedPending"; value: 5 } //ActivityCenterStore.ActivityCenterMembershipStatus.DeclinedPending }
```

Comment:
This is simply not supported inside a `ListElement`; otherwise sure, can still be used within an ordinary model

Reply:
Yes, exactly! The page broke due to using these constants inside `ListElement`, so I simply commented them out by leaving the constant comment so we don't lose track of the values. However, following @caybro's comment, it would probably be worth creating a model like, for example, the one done with `WalletAccountsModel.qml` and then using the constants correctly. I'll create a separate task to do it!

Reply:
Here the task: https://github.com/status-im/status-desktop/issues/12548